bin/clover/src/pipeline-steps/assetSpecificOverrides.ts

```diff
@@ -104,5 +105,95 @@ const overrides = new Map<string, OverrideFn>([
     setAnnotationOnSocket(socket, { tokens: ["Id"] });
 
     variant.sockets.push(socket);
-  }]
+  }],
+  [
+    "AWS::SecretsManager::Secret",
+    addSecretProp("Secret String", "secretString", ["SecretString"]),
+  ],
 ]);
+
+function addSecretProp(
+  secretKind: string,
+  secretKey: string,
+  propPath: string[],
+) {
+  return (spec: ExpandedPkgSpec) => {
+    const variant = spec.schemas[0].variants[0];
+
+    const [secretName] = propPath.slice(-1);
```

Comment:
Makes you wanna pop() eh :) But I imagine this is to make sure you don't mess up the propPath for the caller

Reply:
yup, the JS api for non destructive array operation is annoying but it made no sense to modify it and have something break in the future